xcm/src/v3/junctions.rs

```diff
@@ -221,6 +221,36 @@ impl Junctions {
 		}
 	}
 
+	/// Extract the network ID and the interior consensus location, treating this value as a
+	/// universal location.
+	///
+	/// This will return an `Err` if the first item is not a `GlobalConsensus`, which would indicate
+	/// that this value is not a universal location.
+	pub fn split_global(self) -> Result<(NetworkId, Junctions), ()> {
+		match self.clone().split_first() {
+			(location, Some(Junction::GlobalConsensus(network))) => Ok((network, location)),
+			_ => return Err(()),
```

Comment:
It seems to me that a better API would be to return the original `MultiLocation` if possible, since this function takes ownership of `self` and so the `MultiLocation` disappears afterwards.

On second thought, this might not really matter as `MultiLocation` is now a `Copy` type...